k8s/helm/templates/manager-role.yaml

```
@@ -120,3 +120,14 @@ rules:
     - pods/exec
   verbs:
     - create
+- apiGroups:
+    - ""
+  resources:
+    - events
+  verbs:
+    # used to write events
```

a little bit more precise for beginners :)
```suggestion
    # used to write events while reconciling CRs
```